README.md

```diff
@@ -94,12 +94,25 @@ airflow db init
 Create an SQLite database for the example to run with and run the DAG:
 ```
 sqlite3 /tmp/sqlite_default.db "VACUUM;"
```

Comment:
Why do we need this line? isn't it redundant on mac?

Reply:
Updated to use env var to get file name.